FactorioBlueprintStringRenderer/src/com/demod/fbsr/LogisticGridCell.java

```diff
@@ -179,6 +183,12 @@ public void setMove(Optional<Direction> move) {
 	}
 
 	public void setOutputs(Optional<Set<String>> outputs) {
+		if (outputs.isPresent()) {
+			Set<String> strings = outputs.get();
+			if (strings.contains("UNKNOWN")) {
+				System.out.println("WARNING: outputs contains UNKNOWN");
+			}
+		}

```

Comment:
This logging is probably not necessary anymore and I can remove it. I used it with a breakpoint to find the cause of the warn logging.